libnmstate/state.py

```diff
@@ -76,6 +104,34 @@ def interfaces(self):
         """ Indexed interfaces state """
         return self._ifaces_state
 
+    @property
+    def routes(self):
+        """
+        The list of route in config. Can read and write.
+        """
+        return self._routes
+
+    @routes.setter
+    def routes(self, value):
+        """
+        Replace the routes
+        """
+        self._routes = value
+
+    @property
+    def sorted_routes(self):
+        """
+        The sorted list of route in config. Read only.
+        """
+        return sorted(self._routes, key=_route_sort_key)
+
+    @property
+    def iface_routes(self):
+        """
+        Indexed routes by next hop interface name. Read only.
+        """
+        return State._index_routes_by_iface(self.sorted_routes)
```

Comment:
This seems to be the only user of `sorted_routes` so maybe just sort the routes here instead of making this a public property.